### PR TITLE
symbols: Add spaces in under score and down score

### PR DESCRIPTION
**Pull request by cgoates**
This resolved some recognition issues for me, and seems to follow the general rules of thumb that I've heard to improve recondition.

### CODE REVIEW

**Review by auscompgeek — APPROVED** [review]
Yeah, `down score` should absolutely have a space. `underscore`, whilst it is a real word, if adding a space helps with recognition for people it probably makes sense.

File: text/symbols.talon [hunk 1]
```diff
@@ -1,5 +1,5 @@
 question [mark]: "?"
-(downscore | underscore): "_"
+(down score | under score): "_"
```
Comment (pokey) [inline text/symbols.talon:2]: Ok so I think the problem here is that these appear here and also implicitly via keys.py / keys.talon, with different spacing. This throws off talon's precedence rules. I would just remove this line

Also, given that this one was winning, I'd be tempted to use "underscore" in keys.py

@lunixbochs does that sound reasonable?

**Review by pokey — DISMISSED** [review]

File: text/symbols.talon (hunk 1 above)
Comment (cgoates) [reply text/symbols.talon]: Okay, that makes sense. I made this suggested change and that gets rid of the issue I was having as well as my original solution does. I see a few other duplications between this file and keys.py/keys.talon; is that something that should be cleaned up as well, or is there a reason to keep both around? Mostly just asking for curiosity's sake, though I'm happy to do more here if that is desired.

Comment (auscompgeek) [reply text/symbols.talon]: Ah, of course. Yeah, we realistically shouldn't have duplicated command phrases.

**Review by rntz — APPROVED** [review]

**Comment by rntz** [comment]
ok, so now we just have keys named `underscore` and `down score`. if recognition issues continue we can also add in the variant `under score`.

**Pull request merged**